### ISSUE TITLE
Welcome E-Mail for new users

### ISSUE THREAD

**Issue by daria-sl** [globe-swiss/phaenonet-functions#74]
If new users register on our application, they currently do not get an invitation e-mail. Would it be possible to automatically send such an email to new users @pgoellnitz ? In this message, we would not only welcome them "more personally", but we could also include the links to the video tutorials on how to add an object etc..

**Comment by pgoellnitz** [comment]
Yes, definitely possible. Would need some investigation on best practice and effort involved to implement this.

**Comment by pgoellnitz** [comment]
@daria-sl, this should be pretty straight forward after globe-swiss/phaenonet-client#95
